x/auction/abci.go

```diff
@@ -4,7 +4,8 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
-// BeginBlocker runs at the start of every block.
+// BeginBlocker closes all expired auctions at the end of each block. It panics if
+// there's an error other than ErrAuctionNotFound.
```

Comment:
Given that ErrAuctionNotFound shouldn't happen, any reason not to panic? cc @rhuairahrighairidh